### PR TITLE
email card ue and fe changes

### PR DESCRIPTION
**Pull request by mayukhs2904**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1578

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page
- After: https://feat-exlm-1578--exlm--adobe-experience-league.hlx.page

### CODE REVIEW

File: blocks/email-card/email-card.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    justify-content: space-between;
+    align-items: flex-start;
+    margin-bottom: 16px;
+    gap: 8px;
+}
+
+.email-card-title div {
+    font-size: 12px;
+    font-weight: 700;
+    color: #000000;
```
Comment (noormd08) [inline blocks/email-card/email-card.css:24]: Please get the color code from typography varaibles.

File: blocks/email-card/email-card.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/* mobile */
+.email-card {
+    box-sizing: border-box;
+    max-width: 318px;
+    padding: 24px 24px 32px 24px;
+    background: #FFFFFF 0% 0% no-repeat padding-box;
```
Comment (noormd08) [inline blocks/email-card/email-card.css:6]: Please get the color code from CSS varaibles.

File: blocks/email-card/email-card.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    letter-spacing: 0.72px;
+}
+
+.email-card-title a {
+    display: flex;
+    align-items: center;
+    font-size: 14px;
+    font-weight: 500;
+    letter-spacing: 0px;
+    color: #222222;
```
Comment (noormd08) [inline blocks/email-card/email-card.css:35]: Same as previous comment

File: blocks/email-card/email-card.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    text-transform: uppercase;
+    letter-spacing: 0.72px;
+}
+
+.email-card-title a {
+    display: flex;
+    align-items: center;
+    font-size: 14px;
+    font-weight: 500;
+    letter-spacing: 0px;
```
Comment (noormd08) [inline blocks/email-card/email-card.css:34]: Remove the letter-sapcing.

**Review by ruhisingh1 — DISMISSED** [review]

File: blocks/email-card/email-card.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+const { adobeAccountURL } = getConfig();
+
+let email = '';
+
+const isSignedIn = await isSignedInUser();
+if (isSignedIn) {
+  const profileData = await defaultProfileClient.getMergedProfile();
+  email = profileData?.email;
+}
```
Comment (noormd08) [inline blocks/email-card/email-card.js:24]: Keep these inside the decorate function.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**